### PR TITLE
Fix occurences of plugin double reference in UX

### PR DESCRIPTION
**Pull request by X9VoiD**

### CODE REVIEW

File: OpenTabletDriver.UX/Controls/PluginManager.cs [hunk 1]
```diff
@@ -27,26 +27,23 @@ public PluginManager()
...
             {
-                _plugins = value;
-                _pluginList.Items.Clear();
-                foreach (var plugin in Plugins)
-                    _pluginList.Items.Add(string.IsNullOrWhiteSpace(plugin.Name) ? plugin.Path : plugin.Name);
+                var pluginRef = new PluginReference(type);
+                if (type != typeof(T) && !Plugins.Contains(pluginRef))
+                {
+                    Plugins.Add(pluginRef);
+                }
```
Comment (InfinityGhost) [inline OpenTabletDriver.UX/Controls/PluginManager.cs:36]: Why did you change the `Plugins` set/get when all you really needed to do was check if it was already loaded?

Comment (X9VoiD) [reply OpenTabletDriver.UX/Controls/PluginManager.cs:36]: A bug occurs that if I did the check without changing set/get, the Plugins get cleared, as I have seen on debugger.
Essentially, the check on set/get have to happen after checking that there is no duplicate, not before (which is triggered by L23)

Comment (X9VoiD) [reply OpenTabletDriver.UX/Controls/PluginManager.cs:36]: And when L34 `Plugins = new List<PluginReference>(pluginRefs);` is kept there the plugins still contained the duplicate references.
